.gitlab-ci.yml

```diff
@@ -18,7 +18,7 @@ stages:
 variables:
   # Format: $IMAGE-V$DATE [Cache is not used as of today but kept here
   # for reference]
-  CACHEKEY: "bionic_coq-V2019-03-01-V43"
+  CACHEKEY: "bionic_coq-V2019-03-14-V14"
```

Comment:
It took 14 tries to get something that works?

Reply:
For a while I've been just picking a random number as to avoid possible clashes with other PRs.

Reply:
We could use date-PR number-version